cmake/scripts/linux/platform/X11.cmake

```diff
@@ -0,0 +1,8 @@
+core_require_dep(OpenGl)
+core_require_dep(EGL)
+core_require_dep(X)
+core_require_dep(XRandR)
+core_require_dep(LibDRM)
+core_optional_dep(VAAPI)
+core_optional_dep(VDPAU)
+core_optional_dep(OpenGLES)
```

Comment:
@wsnipex means something like https://github.com/xbmc/xbmc/blob/master/CMakeLists.txt#L103 and https://github.com/xbmc/xbmc/blob/master/CMakeLists.txt#L127

Not a blocker imo.

I've been playing with @fetzerch's work for the last 2 days. Not enough time to finish it, though. I'm very glad you managed to do it!

A big :+1: from me!